### PR TITLE
update VolumeGroupSnapshot to v1beta1

### PR DESCRIPTION
**Pull request by Madhu-1**
Updating the required files to move VGS from alpha to beta API.

### CODE REVIEW

File: scripts/install-snapshot.sh [hunk 1]
```diff
@@ -78,12 +78,12 @@ function create_or_delete_resource() {
 
     if [ "${operation}" == "create" ]; then
         # Argument to add/update
-        ARGUMENT="--enable-volume-group-snapshots=true"
+        ARGUMENT="--feature-gates=CSIVolumeGroupSnapshot=true"
```
Comment (nixpanic) [inline scripts/install-snapshot.sh:81]: this commit message mentions _vsg_ instead of **vgs**, please correct that

**Review by nixpanic — COMMENTED** [review]
Just a nit, all other things look good to me.

**Review by nixpanic — DISMISSED** [review]

**Review by iPraveenParihar — DISMISSED** [review]

**Comment by nixpanic** [comment]
@mergifyio queue

**Comment by nixpanic** [comment]
Fixed a few remaining v8.0.1 references (NFS deployment and helm values.yaml).

**Review by nixpanic — DISMISSED** [review]

**Comment by iPraveenParihar** [comment]
@Madhu-1 

upgrade tests are failing - 
```
  I1213 12:14:48.098122       1 reflector.go:332] Listing and watching *v1alpha1.VolumeGroupSnapshotContent from github.com/kubernetes-csi/external-snapshotter/client/v8/informers/externalversions/factory.go:142
  W1213 12:14:48.099323       1 reflector.go:547] github.com/kubernetes-csi/external-snapshotter/client/v8/informers/externalversions/factory.go:142: failed to list *v1alpha1.VolumeGroupSnapshotContent: the server could not find the requested resource (get volumegroupsnapshotcontents.groupsnapshot.storage.k8s.io)
  E1213 12:14:48.099361       1 reflector.go:150] github.com/kubernetes-csi/external-snapshotter/client/v8/informers/externalversions/factory.go:142: Failed to watch *v1alpha1.VolumeGroupSnapshotContent: failed to list *v1alpha1.VolumeGroupSnapshotContent: the server could not find the requested resource (get volumegroupsnapshotcontents.groupsnapshot.storage.k8s.io)
  ```
  
  install-snapshot.sh installs beta CRDs and upgrade test using 3.12.1 version deploys older snapshotter sidecar.

**Comment by Madhu-1** [comment]
> @Madhu-1
> 
> upgrade tests are failing -
> 
> ```
>   I1213 12:14:48.098122       1 reflector.go:332] Listing and watching *v1alpha1.VolumeGroupSnapshotContent from github.com/kubernetes-csi/external-snapshotter/client/v8/informers/externalversions/factory.go:142
>   W1213 12:14:48.099323       1 reflector.go:547] github.com/kubernetes-csi/external-snapshotter/client/v8/informers/externalversions/factory.go:142: failed to list *v1alpha1.VolumeGroupSnapshotContent: the server could not find the requested resource (get volumegroupsnapshotcontents.groupsnapshot.storage.k8s.io)
>   E1213 12:14:48.099361       1 reflector.go:150] github.com/kubernetes-csi/external-snapshotter/client/v8/informers/externalversions/factory.go:142: Failed to watch *v1alpha1.VolumeGroupSnapshotContent: failed to list *v1alpha1.VolumeGroupSnapshotContent: the server could not find the requested resource (get volumegroupsnapshotcontents.groupsnapshot.storage.k8s.io)
> ```
> 
> install-snapshot.sh installs beta CRDs and upgrade test using 3.12.1 version deploys older snapshotter sidecar.

fixed now, it's a weird situation. we will remove it in next release

**Review by iPraveenParihar — APPROVED** [review]

**Review by yati1998 — APPROVED** [review]

**Review by nixpanic — APPROVED** [review]

**Comment by nixpanic** [comment]
@mergifyio queue

**Pull request merged**